docs/source/user_guide/runtime-conf.md

```diff
@@ -122,6 +122,24 @@ elyra-metadata install runtimes \
 
 Refer to the [Kubeflow Pipelines Configuration settings](#kubeflow-pipelines-configuration-settings) section for an explanation of the parameters. 
 
+#### Exporting runtime configurations
+
+To export runtime configurations:
+
+```bash
+elyra-metadata export runtimes \
+	--schema-name="kfp" \
+	--directory="/tmp/foo"
+```
+
+The above example will export all runtime configurations in the "kfp" schema to the "/tmp/foo/runtimes" directory.
+
+Note that you must specify the `--directory` option. If the `--schema-name` option is not specified, runtime configurations across all schemas will be exported. 
```

Comment:
```suggestion
Note that you must specify the `--directory` option. If the `--schema_name` option is not specified, runtime configurations across all schemas will be exported. 
```

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.